src/js/api/api.js

```diff
@@ -110,7 +111,8 @@ define([
         legacyInit(this);
 
         // These should be read-only model properties
-        this.container = document.createElement('div');
+        this.container = utils.createElement(playerTemplate({}));
+        this.container = this.container.parentElement.removeChild(this.container);
```

Comment:
Not a fan of this. it doesn't make sense to template everything - especially where speed is concerned. The benefit here is setting the element's classname but at the cost of adding a module, and calling and manipulating the result of utils.createElement.

Ultimately we should to do all setup in a document fragment, and append that to the page after the view is ready. We should also remove this responsibility from api.js and let view.js deal with the old container.